pages/Server-Optimizations.md

```diff
@@ -4,11 +4,16 @@
 <summary>Table of Contents</summary>
 
 - [1. Keeping an eye on performance](#1-keeping-an-eye-on-performance)
+  - [a) Server Profiling (/timings)](#a-server-profiling-timings)
+  - [b) Slimefun Profiling (/sf timings)](#b-slimefun-profiling-sf-timings)
+  - [c) Plugin-based Server Profiling](#c-plugin-based-server-profiling)
 - [2. Choosing the right Server Software](#2-choosing-the-right-server-software)
 - [3. Avoiding /reload](#3-avoiding-reload)
 - [4. Disabling backwards-compatibility](#4-disabling-backwards-compatibility)
 - [5. Slower Tick-rates](#5-slower-tick-rates)
 - [6. Handling Cargo networks](#6-handling-cargo-networks)
+  - [a) Setting a maxmimum network size](#a-setting-a-maxmimum-network-size)
+  - [b) Setting a cargo delay](#b-setting-a-cargo-delay)
```

Comment:
:NotLikeThis: Read the doc comment above...

Reply:
I didnt even change these smh

Reply:
The second I edit it back and save it my editor goes BRRR and adds it again, smh VSC why you have to do this

Reply:
minor bump for you svr333, you didnt clear this review.